app/controllers/containers/steps_controller.rb

```diff
@@ -45,9 +45,14 @@ def set_form
           @state.send(:"#{step}") || @state.send(:"build_#{step}"))
     end
 
-    def create_container
+    def create_container(start = true)
       @state.send(:"create_#{step}", params[:"docker_container_wizard_states_#{step}"])
-      container = (service = Service::Containers.new).start_container!(@state)
+      service = Service::Containers.new
+      container = if start.is_a? TrueClass
```

Comment:
@daviddavis, thought the same (https://github.com/theforeman/foreman-docker/pull/103#discussion_r27043575) :)